src/model/otr_model.rs

```diff
@@ -491,4 +493,40 @@ mod tests {
         let scaled_rating = OtrModel::performance_scaled_rating(rating.rating, rating_diff, frequency, scaling);
         assert_abs_diff_eq!(scaled_rating, 990.0);
     }
+
+    #[test]
+    fn test_minimum_rating_maximum_volatility() {
+        let player_ratings = vec![
+            generate_player_rating(1, Osu, 100.1, 500.0, 1),
```

Comment:
May want to use expressions (e.g. `ABSOLUTE_RATING_FLOOR + 0.1`) here instead of just `100.1`, `500.0` so you don't need to change this test in case the constants change

Reply:
Fixed ty